src/main/java/Duke.java

```diff
@@ -1,10 +1,283 @@
+        this.myList = new ArrayList<>();
+    }
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Duke ft = new Duke();
+        ft.init();
+    }
+    public void init() {
+        System.out.println("    ____________________________________________________________\n"
+                + "    Hello! I'm FriendlyTool\n"
+                + "    What can I do for you?\n"
+                + "    ____________________________________________________________"
+        );
+        try {
+            loadTask();
+        } catch (ftException e) {
+            System.out.println(e.getMessage());
+        }
+        Scanner sc = new Scanner(System.in);
+        while (this.isActive) {
+            String input = sc.nextLine();
+            try {
+                nextAction(input);
+            } catch (ftException e) {
+                System.out.println(e.getMessage());
+            }
+        }
+        System.out.println("    ____________________________________________________________\n"
+                + "    Bye. Hope to see you again soon!\n"
+                + "    ____________________________________________________________" );
+    }
+
+    private void nextAction(String input) throws ftException {
+        if (input.isEmpty()) {
+            throw new ftException("Error: Please Type Command");
+        }
+        StringTokenizer st = new StringTokenizer(input);
+        try {
+            CommandTypes command = CommandTypes.valueOf(st.nextToken().toUpperCase());
+            switch (command) {
+                case BYE:
+                    this.isActive = false;
+                    break;
+                case LIST:
+                    showList();
+                    break;
+                case MARK:
+                    mark(st);
+                    break;
+                case UNMARK:
+                    unmark(st);
+                    break;
+                case TODO:
+                    addTask(st, CommandTypes.TODO);
+                    break;
+                case DEADLINE:
+                    addTask(st, CommandTypes.DEADLINE);
+                    break;
+                case EVENT:
+                    addTask(st, CommandTypes.EVENT);
+                    break;
+                case DELETE:
+                    deleteTask(st);
+                    break;
+                default:
+                    throw new ftException("Unknown Command: Please use a correct command");
+            }
+        }catch (IllegalArgumentException e) {
+            throw new ftException("Unknown Command: Please use a correct command");
+        }
+    }
+
+    private void addTask(StringTokenizer st, CommandTypes ct) throws ftException {
```

Comment:
This codefile is a bit long for me, I think it would benefit from more modularization through perhaps a Parser class or something to handle the parsing of the request and task creation. This also reduces the repeated `switch-case` to check task type before acting on the user request